scripts/smoke_w1_real.py

```
@@ -0,0 +1,76 @@
+        mgr_t = BotSessionManager(
+            platform="telegram", identity_resolver=resolver, store=store
+        )
+        mgr_d = BotSessionManager(
+            platform="discord", identity_resolver=resolver, store=store
+        )
+
+        out_t = await mgr_t.chat(
+            agent, "tg-12345",
+            "Remember this: my favourite colour is octarine.",
+        )
+        print(f"\n[Telegram] in:  Remember this: my favourite colour is octarine.")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)